lib/urbanopt/scenario/default_reports/schema/README.md

```diff
@@ -18,17 +18,16 @@ The URBANopt Scenario Gem includes functionality for defining scenarios, running
 
 |            output                        |  unit   |
 | -----------------------------------------| ------- |
-| Electricity:Facility                     | kbtu     |
-| ElectricityProduced:Facility             | kbtu     |
-| Gas:Facility                             | kbtu    |
-| DistrictCooling:Facility                 | kbtu    |
-| DistrictHeating:Facility                 | kbtu    |
+| Electricity:Facility                     | kWh     |
+| ElectricityProduced:Facility             | kWh     |
+| Gas:Facility                             | kWh     |
```

Comment:
Are we really displaying gas in kWh units?

Reply:
Yes,  kWh is energy same as kBtu.

Reply:
you aren't only reporting it in kWh though? kBtu is much more common...was there a particular request for gas (and electric) in kWh?